salt/modules/win_pkg.py

```diff
@@ -1138,15 +1166,18 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
             version_num = six.text_type(version_num)
 
         if not version_num:
+            if pkg_name in old:
+                log.debug('A version ({0}) already installed for package '
+                          '{1}'.format(version_num, pkg_name))
```

Comment:
This log message needs to be fixed in Oxygen, `.format()` changed to string replacement.